src/Compilers/CSharp/Portable/Binder/Semantics/Conversions/Conversion.cs

```diff
@@ -22,7 +22,9 @@ public struct Conversion : IEquatable<Conversion>
         internal static readonly Conversion ImplicitNullable = new Conversion(ConversionKind.ImplicitNullable);
         internal static readonly Conversion ImplicitReference = new Conversion(ConversionKind.ImplicitReference);
         internal static readonly Conversion ImplicitEnumeration = new Conversion(ConversionKind.ImplicitEnumeration);
+        internal static readonly Conversion ImplicitTupleLiteral = new Conversion(ConversionKind.ImplicitTupleLiteral);
         internal static readonly Conversion ImplicitTuple = new Conversion(ConversionKind.ImplicitTuple);
+        internal static readonly Conversion ExplicitTuple = new Conversion(ConversionKind.ExplicitTuple);
```

Comment:
I think it was agreed that we need this. There are enough compelling reasons for this. If I do not find notes for this I will open a proposal at the time I start implementing that. Although I am certain we agreed on doing explicit tuple _type_ conversions. 
I am less certain about explicit tuple conversions from _expression_, but that is another story.